CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+## 3.1.0
+* NEW: Added TypeScript declaration file for compatibility with TypeScript projects. See: https://github.com/getsentry/raven-js/pull/610
```

Comment:
> That, to me, falls under something like, us adding a feature into a plugin.

That should be a minor bump too, yeah. It's an argument for moving these into their own repo.